# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/vserver/no_extraversion.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./vserver-1.24.patch.orig	2003-12-17 11:04:30.000000000 -0300
+++ ./vserver-1.24.patch	2003-12-17 11:04:47.000000000 -0300
@@ -13,18 +13,6 @@
  Micro Memory MM5415 Battery Backed RAM support (EXPERIMENTAL)
  CONFIG_BLK_DEV_UMEM
    Saying Y here will include support for the MM5415 family of
-diff -NurpP --minimal linux-2.4.23/Makefile linux-2.4.23-vs1.22/Makefile
---- linux-2.4.23/Makefile	Fri Nov 28 19:26:21 2003
-+++ linux-2.4.23-vs1.22/Makefile	Sat Dec 13 18:42:20 2003
-@@ -1,7 +1,7 @@
- VERSION = 2
- PATCHLEVEL = 4
- SUBLEVEL = 25
--EXTRAVERSION =
-+EXTRAVERSION = -vs1.26
- 
- KERNELRELEASE=$(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
- 
 diff -NurpP --minimal linux-2.4.23/arch/alpha/kernel/entry.S linux-2.4.23-vs1.22/arch/alpha/kernel/entry.S
 --- linux-2.4.23/arch/alpha/kernel/entry.S	Fri Jun 13 16:51:29 2003
 +++ linux-2.4.23-vs1.22/arch/alpha/kernel/entry.S	Sat Dec 13 18:42:20 2003
